cli/src/plz/cli/controller_proxy.py

```diff
@@ -205,20 +205,25 @@ def get_user_last_execution_id(self, user: str) -> Optional[str]:
             raise ValueError('Expected an execution ID')
 
     def kill_instances(
-            self, instance_ids: Optional[List[str]], force_if_not_idle: bool) \
-            -> bool:
-        instance_ids = instance_ids if instance_ids is not None else []
+            self,
+            user: str,
+            instance_ids: Optional[List[str]],
+            ignore_ownership: bool,
+            including_idle: Optional[bool],
+            force_if_not_idle: bool) -> bool:
         response = self.server.post(
             'instances', 'kill',
             json={
-                'all_of_them_plz': instance_ids == [],
+                'all_of_them_plz': instance_ids is None,
```

Comment:
Not sure I understand this. If I specify instance ids and also 'all_of_them_plz', then it will only kill those instance ids, ignoring 'all_of_them_plz'?. If I haven't specified 'force_if_not_idle' it will only kill idle instances in that case? So 'all_of_them_plz' doesn't necessarily imply 'force_if_not_idle'?

Reply:
Got it, specifying instances also implies force_if_not_idle, correct? In which case, why do we need that argument?

Reply:
Discussed in person.